build/prbuild.yml

```diff
@@ -2,122 +2,165 @@ name: $(date:yyyy-MM-dd)$(rev:.rr)
-      arguments: --no-build --blame --verbosity normal --configuration release
-      command: test
-      projects: |
-        **\*test*.csproj
-
-- job: Debug
-  pool:
-    vmImage: 'windows-2022'
-  steps:
-  - task: NuGetToolInstaller@1
-    displayName: 'Use NuGet 5.x'
-    inputs:
-      versionSpec: '5.x'
-
-  - task: NuGetCommand@2
-    displayName: 'NuGet restore'
-
-  - task: DotNetCoreCLI@2
-    displayName: 'dotnet restore'
-    inputs:
-      command: restore
-      projects: |
-        **\*.csproj
-
-  - task: VSBuild@1
-    displayName: 'Build Solution **\*.sln'
-    inputs:
-      vsVersion: 17.0
-      platform: '$(BuildPlatform)'
-      configuration: debug
-
-  - task: CopyFiles@2
-    displayName: 'Copy Files to: $(Build.ArtifactStagingDirectory)'
-    inputs:
-      Contents: '**\bin\debug\**'
-      TargetFolder: '$(Build.ArtifactStagingDirectory)'
-
-  - task: PublishBuildArtifacts@1
-    displayName: 'Publish Artifact: drop'
-
-  - task: DotNetCoreCLI@2
-    displayName: 'Test Assemblies (.NET Core) **\*test*.csproj'
-    inputs:
-      arguments: --no-build --blame --verbosity normal --configuration debug /p:CollectCoverage=true /p:CoverletOutput=./CodeCoverage/ /p:CoverletOutputFormat=cobertura
-      command: test
-      projects: |
-        **\*test*.csproj
-
-  # Run a command line script using Bash on Linux and macOS and cmd.exe on Windows
-  - task: CmdLine@2
-    displayName: 'Upload coverage to codecov.io'
-    inputs:
-      script: $(USERPROFILE)\.nuget\packages\codecov\$(CODECOV_VERSION)\tools\win7-x86\codecov.exe -t $(CODECOV_TOKEN) -f **/*coverage.cobertura.xml
-    #workingDirectory: # Optional
-    #failOnStderr: false # Optional
-    
-- template: check-dependencies.yml
+  BuildPlatform: "x86"
+  CreateAxeWindowsNugetPackage: "true"
+  CreateAxeWindowsZippedCLI: "true"
+  PublicRelease: "false"
+  SignAppForRelease: "false"
+# The `resources` specify the location and version of the 1ES PT.
+resources:
+  repositories:
+    - repository: 1esPipelines
+      type: git
+      name: 1ESPipelineTemplates/1ESPipelineTemplates
+      ref: refs/tags/release
+extends:
+  # The pipeline extends the 1ES PT which will inject different SDL and compliance tasks.
+  # For non-production pipelines, use "Unofficial" as defined below.
+  # For productions pipelines, use "Official".
+  template: v1/1ES.Unofficial.PipelineTemplate.yml@1esPipelines
   parameters:
-    configuration: release
-  
\ No newline at end of file
+    # Update the pool with your team's 1ES hosted pool.
+    pool:
+      name: $(a11yInsightsPool) # Name of your hosted pool
+      image: windows-2022-secure # Name of the image in your pool. If not specified, first image of the pool is used
+      os: windows # OS of the image. Allowed values: windows, linux, macOS
+    sdl:
+      codeql:
+        compiled:
+          enabled: true
+      binskim:
+        analyzeTargetGlob:
+          "$(System.DefaultWorkingDirectory)\\src\\CI\\bin\\**\\*.dll;\
```

Comment:
Is BinSkim really needed in the PR build? It wasn't in the previous build because the files aren't signed. Looking at the build log, it says that 58 files were scanned, but the results file seems to be empty. If we can, I'd love to omit this step

Reply:
Disabled binskim in unofficial template.